<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rectangle_Dialog</class>
 <widget class="QDialog" name="Rectangle_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>312</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Rectangle options</string>
  </property>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>264</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>264</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>9</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Insert rectangle using</string>
   </property>
   <widget class="QRadioButton" name="_100">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Two corners</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_101">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>44</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Two sides</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>90</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Translate by</string>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>53</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="_102">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Fixed width and height</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="RCDoubleSpinBox" name="_103">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>55</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="RCDoubleSpinBox" name="_104">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>96</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="_105">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>216</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hatch</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="_106">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Square corners</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_100</tabstop>
  <tabstop>_101</tabstop>
  <tabstop>_102</tabstop>
  <tabstop>_103</tabstop>
  <tabstop>_104</tabstop>
  <tabstop>_105</tabstop>
  <tabstop>_106</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
